ceph/arrow-20-compat.patch

```diff
@@ -0,0 +1,877 @@
+From 8da50fe3138e72831efd935d0d41c329974ce760 Mon Sep 17 00:00:00 2001
+From: James Page <james.page@chainguard.dev>
+Date: Tue, 6 May 2025 14:59:48 +0100
+Subject: [PATCH] Refactor crypto integration with arrow
```

Comment:
has this been submitted upstream? Cause I don't think we would want to maintain this in the long term.

Reply:
It has - I can add a reference

Reply:
@xnox added an Origin header to the patch for future travellers.